components/molecule/inputTags/src/config.js

```diff
@@ -12,4 +15,15 @@ export const isDuplicate = (values, newValue) => {
   return upperTags.includes(newValue.toUpperCase())
 }
 
+export const getInputToTagSize = inputSize => {
+  const conversor = {
+    [inputSizes.XSMALL]: atomTagSizes.XSMALL,
+    [inputSizes.SMALL]: atomTagSizes.SMALL,
+    [inputSizes.MEDIUM]: atomTagSizes.MEDIUM,
+    [inputSizes.LARGE]: atomTagSizes.LARGE,
+    [inputSizes.XLARGE]: atomTagSizes.XLARGE
+  }
+  return conversor[inputSize]
```

Comment:
What about a warning on the console if a not allowed size is passed? This could help to identify unexpected behaviours instead of setting the size to `undefined`, which will be the returned value.

Reply:
it is controlled in PropTypes